packages/w3/index.js

```diff
@@ -195,6 +195,24 @@ export async function put (path, opts) {
   console.log(`⁂ https://dweb.link/ipfs/${root}`)
 }
 
+/**
+ * Deletes 1 or more uploads
+ *
+ * @param {string} cid the root CID to fetch
+ * @param {object} opts
```

Comment:
```suggestion
 * @param {object} [opts]
```

let's make opts optional

Reply:
Any reason for that? If so, we should change it on all the commands, no?

Reply:
well, given no one will use this as a lib, I will not expect problems. But typescript compiler would fail if this function is used as `x.deleteUpload(cid)` because `opts` is not documented as optional. However, none of the Object properties are required. Not a big deal here